### PR TITLE
Lots of new mappings

### PR DESCRIPTION
**Pull request by RDIL**
Cool stuff

### CODE REVIEW

File: mappings/net/minecraft/client/render/entity/feature/VillagerResourceMetadata.mapping [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 		FIELD field_17164 name Ljava/lang/String;
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;)V
+			ARG 3 name
 		METHOD method_17168 getName ()Ljava/lang/String;
 		METHOD method_17170 from (Ljava/lang/String;)Lnet/minecraft/class_3888$class_3889;
-	FIELD field_17158 READER Lnet/minecraft/class_3890;
-	FIELD field_17159 hatType Lnet/minecraft/class_3888$class_3889;
-	METHOD method_17167 getHatType ()Lnet/minecraft/class_3888$class_3889;
+			ARG 0 from
+		METHOD values values ()[Lnet/minecraft/class_3888$class_3889;
```
Comment (liach) [inline mappings/net/minecraft/client/render/entity/feature/VillagerResourceMetadata.mapping]:
```suggestion
```

File: mappings/net/minecraft/container/BeaconContainer.mapping [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
+		ARG 2 inventory
+		ARG 4 blockContext
 	METHOD method_17372 setEffects (II)V
 		ARG 1 primaryEffectId
 		ARG 2 secondaryEffectId
 	METHOD method_17373 getProperties ()I
 	METHOD method_17374 getPrimaryEffect ()Lnet/minecraft/class_1291;
 	METHOD method_17375 getSecondaryEffect ()Lnet/minecraft/class_1291;
 	METHOD method_17376 hasPayment ()Z
+	CLASS class_1705 SlotPayment
```
Comment (liach) [inline mappings/net/minecraft/container/BeaconContainer.mapping]: probably rename this to `PaymentSlot`
```suggestion
	CLASS class_1705 PaymentSlot
```

File: mappings/net/minecraft/container/Container.mapping [hunk 3]
```diff
@@ -9,7 +9,7 @@ CLASS net/minecraft/class_1703 net/minecraft/container/Container
 	FIELD field_7762 quickCraftStage I
 	FIELD field_7763 syncId I
 	FIELD field_7764 trackedStacks Lnet/minecraft/class_2371;
-		COMMENT A list of item stacks that is used for tracking changes in {@link #sendContentUpdates()}.
+		COMMENT A list of item stacks that is used for tracking changes in {@link
```
Comment (liach) [inline mappings/net/minecraft/container/Container.mapping]:
```suggestion
		COMMENT A list of item stacks that is used for tracking changes in {@link #sendContentUpdates()}.
```
Think this is a bug with old enigma.

File: mappings/net/minecraft/container/ContainerFactory.mapping [hunk 4]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_1270 net/minecraft/container/ContainerFactory
-	METHOD createMenu (ILnet/minecraft/class_1661;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1703;
+	METHOD createMenu createMenu (ILnet/minecraft/class_1661;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1703;
```
Comment (liach) [inline mappings/net/minecraft/container/ContainerFactory.mapping]:
```suggestion
	METHOD createMenu (ILnet/minecraft/class_1661;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1703;
```
Unfortunatley please don't mark things as deobfuscated. you know, old enigma sucks sometimes.

File: mappings/net/minecraft/container/PlayerContainer.mapping [hunk 5]
```diff
@@ -9,3 +9,4 @@ CLASS net/minecraft/class_1723 net/minecraft/container/PlayerContainer
 		ARG 1 inventory
 		ARG 2 onServer
 		ARG 3 owner
+	METHOD method_7641 emptyArmorSlotIDs ()[Ljava/lang/String;
```
Comment (liach) [inline mappings/net/minecraft/container/PlayerContainer.mapping]:
```suggestion
	METHOD method_7641 getEmptyArmorSlotIds ()[Ljava/lang/String;
```

**Review by liach — CHANGES_REQUESTED** [review]

**Comment by RDIL** [comment]
Thanks @liach for the review!

**Comment by modmuss50** [comment]
Are these backported from newer versions? or are they new to 1.14?

**Comment by RDIL** [comment]
New.  I would make these against master but my mod targets these versions.  I'm fine with others forward-porting these.

**Review by liach — APPROVED** [review]

**Review by Runemoro — APPROVED** [review]

**Comment by RDIL** [comment]
@modmuss50 can we get this merged?

**Comment by RDIL** [comment]
Just a ding to remind people to review/merge :)

**Review by modmuss50 — APPROVED** [review]
Ill merge this in, but like with the other 1.14.4 PR this wont get released as we dont have this setup to release from github actions.

Thanks.

**Pull request merged**